Source/Core/Common/TraversalClient.cpp

```diff
@@ -0,0 +1,374 @@
+// This file is public domain, in case it's useful to anyone. -comex
+
+#include "TraversalClient.h"
+#include "enet/enet.h"
+#include "Timer.h"
+
```

Comment:
Includes are relative to the Core directory. For example, the first one would be `#include "Common/TransversalClient.h"`